### ISSUE TITLE
Test dataset for PlasFlow v1.1.0 not working

### ISSUE THREAD

**Issue by lexming** [easybuilders/easybuild-easyconfigs#9951]
The easyconfig `PlasFlow-1.1.0-foss-2019b-Python-3.7.4.eb` (#9941) seems broken and I think that it was already broken before upgrading its dependency on `TensorFlow` to v2 (#9947). 

Performing the Test Dataset (https://github.com/smaegol/PlasFlow#test-dataset) fails with
```
$ PlasFlow.py --input Citrobacter_freundii_strain_CAV1321_scaffolds.fasta --output test.plasflow_predictions.tsv --threshold 0.7
Importing sequences
Imported  [1] 78
   sequences 
Traceback (most recent call last):
   File "/user/brussel/101/vsc10122/.local/easybuild/software/PlasFlow/1.1.0-foss-2019b-Python-3.7.4/bin/PlasFlow.py", line 92, in <module>
     pd_accessions = pandas2ri.ri2py(accessions) 
AttributeError: module 'rpy2.robjects.pandas2ri' has no attribute 'ri2py'
```
which means that `PlasFlow` is not compatible with `rpy2` v3. The dependency on `rpy2` is indeed listed as for version v2 (`>=2.8`) in the project's github.

**Comment by boegel** [comment]
@Darkless012 Can you take a look at this?

**Comment by Darkless012** [comment]
Will do :)

**Comment by lexming** [comment]
Fix in #10028
